go.mod

```diff
@@ -6,7 +6,7 @@ go 1.21
 replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.6.22 // for compatibilty with docker 20.10.x
+	github.com/containerd/containerd => github.com/containerd/containerd v1.6.27 // for compatibilty with docker 20.10.x
```

Comment:
Does the `// for compatibilty with docker 20.10.x` comment still apply?

Reply:
That's a good question and I don't know the answer. @mattfarina would you know?

The "pinning" in Docker seems to come from:
- https://github.com/moby/moby/blob/v20.10.27/vendor.conf#L137
- https://github.com/moby/moby/blob/v20.10.27/hack/dockerfile/install/containerd.installer#L18

By the way, bumping Docker away from v20.10.x is another needed PR.
-

Reply:
Yeah, we need to rely on docker 20.10.x because there's code in rancher/machine that relies on `pkg/term` (https://github.com/rancher/machine/blob/bc9ca1abb69666ff77a7977d354fa0ae35f92e7b/libmachine/ssh/client.go#L300) and pkg/term was removed with docker 23.0.x. 

Agree that removing dependency on docker v20.10.x would need another PR.

Reply:
To add to what @kinarashah wrote, I'm already working on a change to remove the dependency on docker v20.10.x in Rancher machine. In the future this will hopefully not be an issue.